pyvisa_py/protocols/rpc.py

```diff
@@ -569,14 +579,19 @@ def set_reply_handler(self, reply_handler):
     def set_timeout(self, timeout):
         self.timeout = timeout  # Use None for infinite timeout
 
+    def sock_sendto(self, data, address):
+        ptr = 0
+        while len(data[ptr:]) > 0:
```

Comment:
You could simply write

```suggestion
        while data[ptr:]:
```